DQM/SiStripMonitorCluster/src/SiStripMonitorCluster.cc

```diff
@@ -747,20 +747,20 @@ void SiStripMonitorCluster::analyze(const edm::Event& iEvent, const edm::EventSe
 
       int good_fcc_index = -999;
       for(unsigned int x=0;x<fedConnections.size();x++){
-        //if(fedConnections.at(x)!=NULL && good_fcc_index==-999){
-        if(fedConnections[x]!=NULL && good_fcc_index==-999){
+        if(fedConnections[x]!=NULL){
           good_fcc_index = x;
+          break;
         }
       }
-      if(good_fcc_index!=-999 && fedConnections.at(good_fcc_index)!=NULL){
+      if(fedConnections[good_fcc_index]!=NULL){
```

Comment:
hi @Wilsker - do you need to keep the protection against good_fcc_index-999 here.